pkg/vacuum/vacuum.go

```diff
@@ -338,10 +353,11 @@ func (e *Engine) Run(ctx context.Context) {
 		chunks, err := e.listChunks(ctx, con, w.query)
 		if err != nil {
 			log.Error("msg", fmt.Sprintf("failed to list %s", w.name), "error", err)
+			vacuumErrorsTotal.WithLabelValues(w.workload).Inc()
 			w.stop = true
 			continue
 		}
-		tablesNeedingVacuum.Set(float64(len(chunks)))
+		tablesToVacuum.WithLabelValues(w.workload).Set(float64(len(chunks)))
```

Comment:
If I am understanding this correctly, it seems like the only time the label value other than `locking` is `test` in the unit test. Maybe we declare `locking` out of the functional value and ue that constant everywhere?

Reply:
Not sure I follow. `w.workload` will either be `compressed-chunks-to-freeze` or `compressed-chunks-missing-stats` depending on whether `w` is set to `chunksToFreeze` or `chunksMissingStats`

Reply:
So the label will either be `locking`, `test`, `compressed-chunks-to-freeze`, or `compressed-chunks-missing-stats`